students/hirot_directory/session05/mailroom5.py

```diff
@@ -55,54 +59,30 @@ def quit():
             print("Good bye...")
             sys.exit()
         else:
-            return mainloop()
+          return True # refactoring
 
-    except ValueError as e:
-        print("Error happened. Error is ",type(e),"Please type 1, 2 or 3")
     except KeyError as k:
         print("Error happened. Error is ",type(k),"Please type 1, 2 or 3")
```

Comment:
you don't generally want to report the actual error to the user, simple the fact that they typed an invalid input.